src/components/editor/MonacoEditor/index.tsx

```diff
@@ -131,6 +131,13 @@ function MonacoEditor({
 
     const updateValue = useCallback(
         (isUndo: boolean) => {
+            if (!isUndo && status !== EditorStatus.EDITING) {
+                logRocketConsole('editor:update:skipped', {
+                    status,
+                });
+                return;
+            }
```

Comment:
This should help reduce duplicate updates flowing through the editor. Adding the logging to make sure it is always known when this happens in case there is an issue found.